tests/test_util/aiohttp.py

```diff
@@ -75,8 +75,10 @@ def clear_requests(self):
     @asyncio.coroutine
     # pylint: disable=unused-variable
     def match_request(self, method, url, *, data=None, auth=None, params=None,
-                      headers=None, allow_redirects=None):
+                      headers=None, allow_redirects=None, timeout=None,
+                      json=None):
         """Match a request against pre-registered requests."""
+        data = data or json
```

Comment:
That make it bad for tests... I need revert it on every test case. That will be nonsense